system/default.nix

```diff
@@ -73,12 +98,51 @@
     ];
   };
 
+  # Enable podman with DNS
+  virtualisation.podman = {
+    enable = true;
+  } // (
+    # dnsname allows containers to use ${name}.dns.podman to reach each other
+    # on the same host instead of using hard-coded IPs.
+    # NOTE: --net must be the same on the containers, and not eq "host"
+    # TODO: extend this with flannel ontop of wireguard for cross-node comms
+    if (lib.trivial.release == "22.11") then {
+      defaultNetwork.dnsname.enable = true;
+    } else {
+      defaultNetwork.settings = { dns_enabled = true; };
+    }
+  );
+
+  # Reboots hanged system
   systemd.watchdog.device = "/dev/watchdog";
   systemd.watchdog.runtimeTime = "30s";
 
-  ## Allow passwordless sudo from wheel group
+  # Zram swap
+  zramSwap.enable = true;
+  zramSwap.algorithm = "zstd";
+  zramSwap.memoryPercent = 100;
+
+  # Audit tracing
+  security.auditd.enable = true;
+  security.audit.enable = true;
+  security.audit.rules = [
+    "-a exit,always -F arch=b64 -S execve"
+  ];
```

Comment:
We should figure out what does this mean and does this also work on ARM devices. The `arch=b64` is a bit alarming flag to me.

Reply:
I recall noticing some audit errors on `ponkila-gamma`'s journalctl.

Reply:
I guess we can then just keep it for now.